src/components/EventDetails/EventDetails.js

```diff
@@ -42,14 +39,16 @@ class EventDetails extends Component {
     this.state = {
       isFetchingEvent: true,
       event: null,
-      socialBtnsDisplay: styles.socialButtonsHide
+      socialPopupOpen: false
     };
 
-    this.showHideSocialBtns = this.showHideSocialBtns.bind(this);
+    this.toggleSocialPopup = this.toggleSocialPopup.bind(this);
   }
 
   componentDidMount() {
     const { eventId } = this.props.match.params;
+    window.addEventListener('click', this.handleDocumentClick.bind(this));
```

Comment:
I made this same mistake a few commits ago -- you have to do the `.bind(this)` once in the constructor and use the same exact function in the addEventListener as you do the removeEventListener because every time you call `.bind(this)` you're returning a new/different function (object).